device_hints.c

```diff
@@ -0,0 +1,22 @@
+
+#include <stdint.h>
+
+typedef struct {
+    char* path;
+    uint64_t iomem_start;
+    uint64_t iomem_end;
+    uint32_t irq;
+} devhint_t;
+
+devhint_t hints[] = {
+{ 	.iomem_start = 0x3f8,
+	.irq = 0x4,
+	.iomem_end = 0x3ff,
+	.path = "/rootdev/pci@0/isab@0/isa@0/uart@0" },
+{ 	.iomem_start = 0x2f8,
+	.irq = 0x3,
+	.iomem_end = 0x2ff,
+	.path = "/rootdev/pci@0/isab@0/isa@0/uart@1" },
+
```

Comment:
We should match the names of the devices / drivers with the `.path` string in some way.

Reply:
We have to distinguish resources placed in IOMEM space and IOPORT space. See `include/mips/malta.h`.

Reply:
>We have to distinguish resources placed in IOMEM space and IOPORT
Valid note, done! 
>We should match the names of the devices / drivers with the .path string in some way.
Yes, we should, but I think that it is out of the scope of this PR